packages/Popover/src/Popover.js

```diff
@@ -441,13 +431,14 @@ class Popover extends React.Component {
       this.props.shouldKeepFocus,
       this.state.tip,
       zIndex,
-      this.ariaIdForContent
+      this.ariaIdForContent,
+      this.$trigger
     );
 
     return (
       <ThemeContext.Provider value={isDark}>
         <PopoverContext.Provider value={contextValue}>
-          <sc.Popover data-pka-anchor="popover" {...moreProps} ref={this.$popover}>
+          <sc.Popover data-pka-anchor="popover" {...moreProps} ref={this.$popover} onKeyDown={this.handleKeyDown}>
```

Comment:
🤔 I can't find if it's allowed or not to use `onKey` event without using a tabIndex or aria roles attributes, also wondering if this can be hurtful if someone wants to take control of the Popover onKeyDown event over the Popover, with this approach is not possible.

Also if you decided to go with this approach maybe would be better to use onKeyUp.

Reply:
I was getting rid of the `keyDown` listener on document object and putting it on the component, thereby not need to do event cleanup (https://github.com/acl-services/paprika/pull/987/files#diff-881a787b5655895f8685a371e2d800027c5b395cad9574a2faf72dea1e418287L404) as unmounting will take care of this...